### PR TITLE
W-13586749-new-flag-fa

### PR DESCRIPTION
**Pull request by failup**

### CODE REVIEW

File: modules/ROOT/pages/_partials/cloudhub2-apps.adoc [hunk 1]
```diff
@@ -63,11 +63,12 @@ Deploys the application specified in `<appID>` to the deployment target specifie
...
-|`deploymentTargetId` |ID of the deployment target. +
-You can get this ID directly from Runtime Manager.  | `cloudhub-us-east-1`
-|`name` |Name of the application to deploy. | `testcloudhub2app`
-|`runtimeVersion` |Runtime version of the deployment target.|`4.4.0`
+|`--artifactId` |Artifact ID of the application retrieved from Exchange. | `--artifactId mule-test-plugin`
+|`--deploymentTargetId` |ID of the deployment target. +
+You can get this ID directly from Runtime Manager.  | `--deploymentTargetId cloudhub-us-east-1`
+|`--name` |Name of the application to deploy. | `--name testcloudhub2app`
+|`--runtimeVersion` |Runtime version of the deployment target.|`--runtimeVersion 4.4.0`
+|`--secureProperty` | Sets an encripted property. | `--secureProperty camtest2:3 d1519c7b-16ac-45f3-a771-5bcdcf23dda4`
```
Comment (tasmoore) [inline modules/ROOT/pages/_partials/cloudhub2-apps.adoc:71]:
```suggestion
|`--secureProperty` | Sets an encrypted property. | `--secureProperty camtest2:3 d1519c7b-16ac-45f3-a771-5bcdcf23dda4`
```

**Review by tasmoore — APPROVED** [review]

**Review by tasmoore — APPROVED** [review]

File: modules/ROOT/pages/_partials/cloudhub2-apps.adoc [hunk 2]
```diff
@@ -63,11 +63,12 @@ Deploys the application specified in `<appID>` to the deployment target specifie
...
-|`deploymentTargetId` |ID of the deployment target. +
-You can get this ID directly from Runtime Manager.  | `cloudhub-us-east-1`
-|`name` |Name of the application to deploy. | `testcloudhub2app`
-|`runtimeVersion` |Runtime version of the deployment target.|`4.4.0`
+|`--artifactId` |Artifact ID of the application retrieved from Exchange. | `--artifactId mule-test-plugin`
+|`--deploymentTargetId` |ID of the deployment target. +
+You can get this ID directly from Runtime Manager.  | `--deploymentTargetId cloudhub-us-east-1`
+|`--name` |Name of the application to deploy. | `--name testcloudhub2app`
+|`--runtimeVersion` |Runtime version of the deployment target.|`--runtimeVersion 4.4.0`
+|`--secureProperty` | Sets an encrypted property. | `--secureProperty camtest2:3 d1519c7b-16ac-45f3-a771-5bcdcf23dda4`
```
Comment (alerapo) [inline modules/ROOT/pages/_partials/cloudhub2-apps.adoc]: can we make it more general ?

Comment (alerapo) [reply modules/ROOT/pages/_partials/cloudhub2-apps.adoc]: like:

`--secureProperty

Sets a secure property.
Format: name:value

Example: --secureProperty secureTestProperty:true`

**Review by dmerlob — APPROVED** [review]

**Review by alerapo — COMMENTED** [review]
LGTM

**Pull request merged**